wremnants/datasets/datasetDict2017_v9.py

```diff
@@ -4,16 +4,16 @@
 lumijson = f"{common.data_dir}/Cert_294927-306462_13TeV_UL2017_Collisions17_HLT_IsoMu24_v_CustomJSON.txt"
 
 # NOTES
-# BASE_PATH is /scratchnvme/wmass/NANOV9/postVFP (so 2017 BASE path is {BASE_PATH}/../y2017/) have to update at some point
+# BASE_PATH is /scratchnvme/wmass/NANOV9/postVFP (so 2017 BASE path is {BASE_PATH}/../y2017v6/) have to update at some point
 # ZtautauPostVFP sample is one available from centrl production, so
 dataDictV9_2017 = {
     "dataPostVFP": {
```

Comment:
This is not content of this PR but why is everything labeled with PostVFP? PostVFP refers to the 2016 data set. Can it be changed to 2017 everywhere? And 2018 in the 2018 file?